### PR TITLE
add functionality for gameplay moments on the highlight section

### PR DESCRIPTION
**Pull request by Alexandru-Bulai**
# PR 28-04-24 : Gameplay imgs
## Overview
In this PR we add the images of gameplay moments on the highlight section 
## Key Changes :
 1) index.js : 
     - We add a new function `getImage()`  that is going to get through the  images from the json file and display them one by one every second and a half 
     - Added comments to understand better what each section is about 

Mar-28-2024 14-37-35

### CODE REVIEW

**Review by vladiiancu — COMMENTED** [review]
I would like to challenge you on most of the comments. 

If a variable is called pageCount and your comment is.. a variable that store the count of pages.. isn't it redundant? 

If you actually need the comment, then the name of variable is not good enough. 

My recommendation would be to change the name so that it is crystal clear without any sort of comments. 

If it's clear already l, you don't need the comments at all so they can be dropped. Or in this case, not added

File: src/javaScript/indexDom.js [hunk 1]
```diff
@@ -103,35 +107,51 @@ function populateGamesContainer (payload, gamesContainer) {
...
+  if (Array.isArray(game.gameplay) && game.gameplay.length > 0) {
+    const gameplayImages = `<img class="w-full h-full gameplay-image-${index}" src="/assets/home-page/gameplay/${game.gameplay[currentIndex]}" alt="${game.name} gameplay moment"/>`
+
+    const updateImage = () => {
+      currentIndex = (currentIndex + 1) % game.gameplay.length
+      const imgElement = document.querySelector(`.gameplay-image-${index}`)
+      imgElement.src = `/assets/home-page/gameplay/${game.gameplay[currentIndex]}`
+    }
+
+    setInterval(updateImage, 2000)
```
Comment (vladiiancu) [inline src/javaScript/indexDom.js:151]: Unclear to me why is the setInterval needed.

Comment (Alexandru-Bulai) [reply src/javaScript/indexDom.js:151]: i dont have space to put all the gameplays in there so i rotate them when the user hovers over the container... 
Should i not do it?

Comment (vladiiancu) [reply src/javaScript/indexDom.js:151]: Is this valid for all screen sizes?

**Review by vladiiancu — APPROVED** [review]

**Pull request merged**